packages/config/config/devices/templates/master_template.json

```diff
@@ -164,6 +164,26 @@
 		"writeOnly": false,
 		"allowManualEntry": true
 	},
+	"base_true_false": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 1,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "False",
+				"value": 0
+			},
+			{
+				"label": "True",
+				"value": 1
+			}
+		]
```

Comment:
I must have missed that - where do we need True/False instead of Enable/Disable?

Reply:
In zw100. There are a series of readonly parameters reporting whether the device is above/below various limits.